core/controllers/acl_decorators_test.py

```diff
@@ -1791,7 +1793,11 @@ def get(self) -> None:
 
     def setUp(self) -> None:
         super().setUp()
-        self.signup(feconf.SYSTEM_EMAIL_ADDRESS, self.CURRICULUM_ADMIN_USERNAME)
+        self.system_email_address = (
+            platform_parameter_services.get_platform_parameter_value(
+                platform_parameter_list.ParamName.SYSTEM_EMAIL_ADDRESS.value))
+        assert isinstance(self.system_email_address, str)
```

Comment:
This is a test, so everything here should be deterministic. You can set the platform parameter value to what you want it to be, or mock it -- don't rely on what happens to be in get_platform_parameter_value() unless you are testing the behaviour of that function.

If you want to verify that the default value is a string then you can check that here in setUp() ... it doesn't really need to be checked in every test. (But why do we need to validate that it's a string in the first place ... why is ambiguity even arising here?)

Reply:
Thanks, updated. The usage of `get_platform_parameter_value` is removed from tests.

The string type check was required because the return type of `get_platform_parameter_value` is `platform_parameter_domain.PlatformDataTypes` which is `Union[str, int, bool, float]`.